lib/src/gestures/flutter_map_interactive_viewer.dart

```diff
@@ -329,6 +330,16 @@ class FlutterMapInteractiveViewerState
     --_pointerCounter;
     crDegrees = crDragDegrees;
 
+    // If the keyboard/cursor rotation did not drag, then just set north = cursor
+    if (crRotationTriggered.value && crInitialDegrees == _camera.rotation) {
```

Comment:
Do we want to check for this condition as well before entering this if statement?
```
_interactionOptions.cursorRotationBehaviour == CursorRotationBehaviour.setNorth
```

Reply:
I guess we can, but I did intentionally leave it at this, as it feels like better UX to me in both modes.